formats/finder/frontend/examples/tbd-service-assessments.json

```diff
@@ -0,0 +1,36 @@
+{
+  "content_id": "68518b2a-bb48-4807-99d6-de31d39b3810",
+  "base_path": "/tbd-service-assessments",
+  "title": "All Service Assessments and Self Certifications",
+  "description": "A list of service assessments and self certifications",
+  "need_ids": [
+
+  ],
+  "locale": "en",
+  "updated_at": "2015-07-21T12:45:18.202Z",
+  "public_updated_at": "2015-04-29T09:22:41.000+00:00",
+  "details": {
+    "facets": [
+
+    ],
+    "document_noun": "report",
+    "filter": {
+      "document_type": "assessment_report"
+    },
+    "format_name": "Service Assessment and Self Certification",
+    "show_summaries": true
+  },
+  "links": {
+    "organisations": [
+
+    ],
+    "related": [
+
+    ],
+    "available_translations": [
```

Comment:
`available_translations` are always present, the publishing-api adds it. At the least it should contain itself.